MinecraftClient/Resources/lang/en.ini

```
@@ -310,6 +310,7 @@ cmd.move.moving=Moving {0}
 cmd.move.dir_fail=Cannot move in that direction.
 cmd.move.walk=Walking to {0}
 cmd.move.fail=Failed to compute path to {0}
+cmd.move.suggestforce=The implemented protection prevents a movement in this direction. Use the -f parameter to force the movement.
```

The path either does not exists _or_ is unsafe. So the message could look like this instead:
```suggestion
cmd.move.suggestforce=Failed to compute a safe path to {0}. Try -f parameter to allow unsafe movements.
```